source/transactions/tests/retryable-abort-errorLabels.yml

```diff
@@ -93,7 +93,7 @@ tests:
       - name: abortTransaction
         object: session0
         result:
-          errorLabelsContain: ["RetryableWriteError"]
+          errorLabelsOmit: ["RetryableWriteError"]
```

Comment:
This relates to my earlier comment about `errorLabels: []` in the fail point suppressing the label for a code that is otherwise retryable. Perhaps we should add a comment after `[]` as well. Feel free to copy it around the various places where this applies.

Reply:
Sorry to be clear -- are you asking me to add a comment after `errorLabels: []` or after the line that you've commented on here?

Reply:
A comment after `errorLabels: []` in the fail point would be helpful. This was referring to this bit of text from https://github.com/mongodb/specifications/pull/672#discussion_r338222564:

> Do you think it's worth having a comment after `[]` here to note that we're suppressing the label for an error code that would otherwise be considered retryable?

Reply:
I have added comments after all instances of `errorLabels: []` as well as comments clarifying the expectations of each test.